.gitignore

```diff
@@ -76,3 +76,579 @@ GTestBase.*
 /Fw/Python/src/fprime.egg-info
 /Gds/src/fprime_gds.egg-info
 **/DefaultDict/serializable/*
+/.idea/inspectionProfiles/profiles_settings.xml
+/.idea/inspectionProfiles/Project_Default.xml
+/.idea/.gitignore
+/.idea/fprime.iml
+/.idea/misc.xml
+/.idea/modules.xml
+/.idea/vcs.xml
+/venv/bin/activate
```

Comment:
wouldn't it make a lot more sense to just ignore the entire `/venv/` directory?

Reply:
Yes, I will fix this in next commits

Reply:
I pushed a commit to fix this issue.